src/servers/src/http/handler.rs

```diff
@@ -39,7 +41,9 @@ pub async fn sql(
     let sql_handler = &state.sql_handler;
     let start = Instant::now();
     let resp = if let Some(sql) = &params.sql {
-        JsonResponse::from_output(sql_handler.do_query(sql).await).await
+        // TODO(LFC): Sessions in http server.
+        let session_ctx = Arc::new(SessionContext::new());
```

Comment:
We'd better make http request-scoped and avoid to introduce session concept, which makes deployment complex.

Reply:
An http request can have its own `Context` to store tenant information and other settings, but to be precise it's not a *session* as in traditional web development.

Reply:
WDYT if rename `SessionContext` to `QueryContext`? cc @killme2008

Reply:
DataFusion has a struct named `SessionContext` acts as the context during query execution [here](https://docs.rs/datafusion/latest/datafusion/execution/context/struct.SessionContext.html). I prefer to avoid duplicate naming here.

Reply:
`QueryContext` sounds good to me.